React/creating-components-2.md

```diff
@@ -2,81 +2,73 @@
-
-1. Locate the spot where the `Student` component is rendered from the `App` component.
-
-1. Update the component rendering to include two new parameters by adding them in like this:  
-    - `<Student name="Improved Ada" email="improved-ada@ada.co" />`
-    - This shouldn't change anything yet in our application, so save the file and refresh the browser to verify that the data still looks like the original hard-coded data.
-
-1. Update the `render` function in the `Student` component to replace the hard-coded values with code tags `{ }` which contains the `prop` that came in from the parent component.  
-    - It should now contain `{ this.props.name }` and `{ this.props.email }`
-    - Verify that the content displayed is now coming from the values in the `App` component
-
-
-If we identify each individual piece of the component rendered, we'll see:
-![component prop breakdown](images/component-prop-breakdown.png)
-
-Prop names can be any variable you want them to be. In this particular case, we chose `name` and `email` to store our values.
-
-Once the prop names are **passed in** to the component, we can then use them within the component. That's where `this.props` comes in!
-
-## Setup to manage list of data
-One student is great and all, but ideally, we'd really be able to manage our whole list of students that we might need.
+### Setup to manage list of data
+One student is great and all, but ideally, we'd really be able to manage our whole list of students that we might need. We'd also like to do this _outside_ of the `App` component, because what if our application also tracks Teachers?
 
 We'll create a new component called `StudentCollection` that will live between the existing  `App` and `Student` components. This new component will manage the full list of students and then render the data for each student.
 
-Spend a few minutes now going back to the [creating components](creating-components.md) notes to see how to create the new component file with the basic class, imports and exports you need.
+1. Spend a few minutes now going back to the [creating components](creating-components.md) notes to see how to create the new component file with the basic class, imports and exports you need.
 
-Move the code that renders `Student` from `App` to `StudentCollection`. Note: You will also need to move and update the `import` statement.
+1. Move the code that renders each `Student` from `App` to `StudentCollection`. Note: You will also need to move and update the `import` statement.
 
-Import and render `StudentCollection` now from the `App` component instead.
+1. Import and render `StudentCollection` now from the `App` component instead. Note: You will need to add an import statement.
 
 ## Use `state` for data
+One way that we often set initial `state` data in a component, is by taking data passed in through `props` (from the parent) and setting that to `state` in our constructor. Let's see an example!
```

Comment:
It doesn't look like we show an example of a component's constructor function taking (or using) `props` as its parameter.

It's probably best to just mention it here even if we can't show an example (it doesn't seem like there's an easy way to provide an example of that here).

Reply:
I think we only need that in a functional stateless component, so maybe we should just include it there